sources/Camera/Camera.hpp

```diff
@@ -0,0 +1,29 @@
+//
+// Created by Tom Augier on 2021-05-20.
+// Edited by Benjamin Henry on 2021-05-20.
+// Edited by Louis Auzuret on 2021-05-20.
+//
+
+#pragma once
+
+#include <string>
+#include "lib/wal/sources/Entity/Entity.hpp"
+#include "lib/wal/sources/Models/Vector3.hpp"
+
+class Camera : public WAL::Entity
```

Comment:
You should not create custom entities. If you want to add functionalities to an entity, add a component to the entity. This is why this is called an ECS.